application/controllers/rest/V1/QuestionController.php

```diff
@@ -0,0 +1,434 @@
+use LimeSurvey\Api\Command\V1\QuestionPropertiesSet;
+use LimeSurvey\Api\Command\V1\QuestionDelete;
+
+/**
+ * Question Controller
+ *
+ */
+class QuestionController extends LSYii_ControllerRest
+{
+    /**
+     * Create question.
+     *
+     * @OA\Post(
+     *      path="/rest/v1/question",
+     *      security={{"bearerAuth":{}}},
+     *      summary="Create question",
+     *      description="Create question",
+     *      tags={"Question"},
+     *      @OA\RequestBody(
+     *          @OA\JsonContent(
+     *              required={
+     *                  "surveyID",
+     *                  "groupID",
+     *                  "newQuestionTitle",
+     *                  "newQuestion"
+     *              },
+     *              @OA\Property(
+     *                  property="surveyID",
+     *                  type="string"
+     *              ),
+     *              @OA\Property(
+     *                  property="groupID",
+     *                  type="string"
+     *              ),
+     *              @OA\Property(
+     *                  property="importData",
+     *                  type="string"
+     *              ),
+     *              @OA\Property(
+     *                  property="importDataType",
+     *                  type="string"
+     *              ),
+     *              @OA\Property(
+     *                  property="newQuestionTitle",
+     *                  type="string"
+     *              ),
+     *              @OA\Property(
+     *                  property="newQuestion",
+     *                  type="string"
+     *              ),
+     *              @OA\Property(
+     *                  property="newQuestionHelp",
+     *                  type="string"
+     *              ),
+     *              example={
+     *                  "surveyID": "1",
+     *                  "groupID": "1",
+     *                  "newQuestionTitle": "In what country do you live?",
+     *                  "newQuestion": "In what country do you live?",
+     *                  "newQuestionHelp": "In what country do you live?"
+     *              }
+     *          )
+     *      ),
+     *      @OA\Response(
+     *      response="200",
+     *          description="Success",
+     *          @OA\JsonContent(
+     *              @OA\Schema(
+     *                  type="string",
+     *                  example=123
+     *              )
+     *          )
+     *      ),
+     *      @OA\Response(
+     *          response="400",
+     *          description="Bad request",
+     *          @OA\JsonContent(
+     *              ref="#/components/schemas/response_status"
+     *          )
+     *      ),
+     *      @OA\Response(
+     *          response="401",
+     *          description="Unauthorized"
+     *      )
+     * )
+     *
+     * @return void
+     */
+    public function actionIndexPost()
+    {
+        $request = Yii::app()->request;
```

Comment:
Why can't we pass the Yii request object directly to the command?

Reply:
Because getParam is not compatible with getRestParams, since request can come as json blob.